common/app/renderers/DotcomRenderingService.scala

```diff
@@ -135,6 +138,7 @@ class DotcomRenderingService extends Logging {
         case 200 =>
           Cached(page)(RevalidatableResult.Ok(Html(response.body)))
             .withHeaders("X-GU-Dotcomponents" -> "true")
+            .withPreconnect(HttpPreconnections.defaultUrls)
```

Comment:
I think this is a good start, but it'll be something we need to maintain/optimise carefully for DCR specifically in case there is any divergence on the client with what we load.

Reply:
Good idea.